index.html

```diff
@@ -65,12 +66,12 @@ <h1>Hello, world!</h1>
       <!-- Example row of columns -->
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
+          <h2>Poets Link</h2>
           <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <p><a class="btn btn-default" href="https://www.poetryfoundation.org/" role="button">View details &raquo;</a></p>
```

Comment:
I think you should change the text from view details to something related to poet links.